modules/installation-load-balancing-user-infra.adoc

```diff
@@ -145,7 +145,7 @@ ifdef::user-managed-lb[]
 This section provides an example API and application Ingress load balancer configuration that meets the load balancing requirements for clusters that are deployed with user-managed load balancers. The sample is an `/etc/haproxy/haproxy.cfg` configuration for an HAProxy load balancer. The example is not meant to provide advice for choosing one load balancing solution over another.
 endif::user-managed-lb[]
 
-In the example, the same load balancer is used for the Kubernetes API and application ingress traffic. In production scenarios, you can deploy the API and application ingress load balancers separately so that you can scale the load balancer infrastructure for each in isolation.
+In the example, the cluster name is `ocp4` and the base domain is `example.com`. Here, the same load balancer is used for the Kubernetes API and application ingress traffic. In production scenarios, you can deploy the API and application ingress load balancers separately so that you can scale the load balancer infrastructure for each in isolation.
```

Comment:
Might just be me, but "Here," feels a little strange or formal. Just a few alternative suggestions, if you like:

```suggestion
In the example, the cluster name is `ocp4`, the base domain is `example.com`, and the same load balancer is used for the Kubernetes API and application ingress traffic. In production scenarios, you can deploy the API and application ingress load balancers separately so that you can scale the load balancer infrastructure for each in isolation.
```

Maybe easier to scan:

```suggestion
This example specifies the following:

* The cluster name is `ocp4`.
* The base domain is `example.com`.
* The same load balancer is used for the Kubernetes API and application ingress traffic.

In production scenarios, you can deploy the API and application ingress load balancers separately so that you can scale the load balancer infrastructure for each in isolation.
```
Also likely that I'm overthinking it and it's fine as-is. Up to you. 😅

Reply:
Committing these changes :)